BuildNotifications/ViewModel/Settings/SettingsViewModel.cs

```diff
@@ -120,9 +103,55 @@ public void UpdateUser()
             }
         }
 
-        private readonly IConfiguration _configuration;
+        private void EditConnections()
+        {
+            EditConnectionsRequested?.Invoke(this, EventArgs.Empty);
+        }
+
+        private void ImportExport()
+        {
+            _popupService.ShowImportExportPopup(_configurationService);
+            UpdateOptions();
```

Comment:
After an import BN should to a hard refresh. Just like every time the connections are edited.

Reply:
Otherwise you could import settings and not see the changes take effect after opening and closing the "Edit connections" view